src/main/webapp/postScript.js

```diff
@@ -15,7 +15,7 @@ function fetchBlobstoreUrlAndShowForm() {
       });
 }
 
-function getPosts(userId) {
```

Comment:
It looks like `deleteSinglePost()` still calls `getPosts()`, which appears to be deleted here. Is this intentional?